src/all/peppercarrot/src/eu/kanade/tachiyomi/extension/all/peppercarrot/PepperCarrotFactory.kt

```diff
@@ -0,0 +1,261 @@
+}
+
+class PepperCarrotEO : PepperCarrot("eo") {
+    override val name = "Pipron kaj Karoĉjon"
+
+    override val description = "Libera kaj malfermkoda retkomikso subtenata rekte de ĝiaj legantoj por ŝanĝi la sektoron de bildrakontoj!"
+}
+
+class PepperCarrotVI : PepperCarrot("vi") {
+    override val name = "Hạt Tiêu & Cà Rốt"
+
+    override val description = "Một bộ truyện tranh hoàn toàn miễn phí/tự do được ủng hộ trực tiếp bởi độc giả để thay đổi nền công nghiệp truyện tranh!"
+}
+
+class PepperCarrotNN : PepperCarrot("nn") {
+    override val name = "Pepar & Gulrot"
+
+    override val description = "Ein fri nettserie, med opne kjelder, finansiert direkte av lesarane i håp om å endra teikneserie bransjen!"
+}
+
+class PepperCarrotGD : PepperCarrot("gd") {
+    override val name = "Peabar ⁊ Curran"
+
+    override val description = "Dealbh-èibhinn saor is le bun-tùs fosgailte le taic dìreach leis na pàtranaich aige ach an atharraich sinn gnìomhachas nan dealbhan-èibhinn!"
+}
+
+// more languages with metadata translations
+
+class PepperCarrotDE : PepperCarrot("de") {
+    override val description = "kostenloser, freier und quelloffener Webcomic, direkt durch seine Förderer unterstützt, um die Comic-Industrie zu verändern!"
+}
+
+class PepperCarrotLS : PepperCarrot("es-419", "ls") {
+    override val description = "¡Un webcomic de cultura abierta y código libre apoyado directamente por sus patrocinadores para cambiar la industria del cómic!"
+}
+
+// languages without metadata translations
+
+class PepperCarrotAR : PepperCarrot("ar")
+
+class PepperCarrotBN : PepperCarrot("bn")
+
+class PepperCarrotBR : PepperCarrot("br")
+
+class PepperCarrotCA : PepperCarrot("ca")
+
+class PepperCarrotCS : PepperCarrot("cs")
+
+class PepperCarrotDA : PepperCarrot("da")
+
+class PepperCarrotEL : PepperCarrot("el")
+
+class PepperCarrotFA : PepperCarrot("fa")
+
+class PepperCarrotFI : PepperCarrot("fi")
+
+class PepperCarrotHE : PepperCarrot("he")
+
+class PepperCarrotHI : PepperCarrot("hi")
+
+class PepperCarrotHU : PepperCarrot("hu")
+
+class PepperCarrotID : PepperCarrot("id")
+
+class PepperCarrotIE : PepperCarrot("ie")
+
+class PepperCarrotIO : PepperCarrot("io")
+
+class PepperCarrotIT : PepperCarrot("it")
+
+class PepperCarrotJB : PepperCarrot("jbo-Latn", "jb")
+
+class PepperCarrotJZ : PepperCarrot("jbo", "jz")
+
+class PepperCarrotKH : PepperCarrot("ko-Hani", "kh")
+
+class PepperCarrotKR : PepperCarrot("ko", "kr")
+
+class PepperCarrotKT : PepperCarrot("avk", "kt")
+
+class PepperCarrotKW : PepperCarrot("kw")
+
+class PepperCarrotLA : PepperCarrot("la")
+
+class PepperCarrotML : PepperCarrot("ml")
+
+class PepperCarrotMS : PepperCarrot("ms")
+
+class PepperCarrotNL : PepperCarrot("nl")
+
+class PepperCarrotNO : PepperCarrot("nb", "no")
+
+class PepperCarrotNS : PepperCarrot("nds", "ns")
+
+class PepperCarrotPH : PepperCarrot("fil", "ph")
+
+class PepperCarrotPT : PepperCarrot("pt")
```

Comment:
It's actually `pt-BR` by looking at the text.

```suggestion
class PepperCarrotPT : PepperCarrot("pt-BR", "pt")
```

Reply:
I think I'll keep it as `pt` since its `iso_locale` isn't specified [here](https://www.peppercarrot.com/0_sources/langs.json).

Reply:
It's wrong there then, as it's clearly pt-BR and not pt.

Reply:
Did you check all the chapters? It could be inconsistent since there's multiple translators.

Reply:
I checked a few ones.

Reply:
Checked all now, besides some silent chapters, all of them are indeed in pt-BR.

Reply:
In that case, I'll submit a merge request upstream to fix it there first.

Reply:
Can already fix it in thr extension since changing it later will requiring overriding the id because of possible existing users. Would be better to avoid that as much as we can.